Add emodel-generalisation (#11)
BlueBrain/singlecell-emodel-suite: README.md

- [Currentscape](#currentscape) - Plot currents in electrical models
- [SSCxEModelExamples](#SSCxEModelExamples) - Reproduce Reva, Maria, et al. "A universal workflow for creation, validation and generalization of detailed neuronal models" (2023).
- [e-model-packager](#e-model-packager) - Creates e-model packages from circuits

> - [emodel-generalisation](#emodel-generalisation) - Generalisation of electrical models of neurons with MCMC

Reviewer: ```suggestion
- [emodel-generalisation](#emodel-generalisation) - Generalisation and MCMC sampling of electrical models of neurons
```

Useful links:
[GitHub repo](https://github.com/BlueBrain/e-model-packager),
[Documentation](https://e-model-packager.readthedocs.io/en/latest/).

## emodel-generalisation

> Generalisation of neuronal electrical models on a morphological population with Markov Chain Monte-Carlo.

Reviewer: ```suggestion
Generalisation and MCMC sampling of neuronal electrical models.
```

Useful links:
[GitHub repo](https://github.com/BlueBrain/emodel-generalisation),

> [Documentation](https://emodel-generalisation.readthedocs.io/en/latest/).

Reviewer 2: I think you can also add the link and [citation](https://github.com/BlueBrain/emodel-generalisation?tab=readme-ov-file#emodel-generalisation) of the paper here.
